enabler/src/main/java/com/openxc/enabler/preferences/UploadingPreferenceManager.java

```diff
@@ -61,17 +61,18 @@ private void setUploadingStatus(boolean enabled) {
                 stopUploading();
             }
 
-            while (!getVehicleManager().isViConnected()) {
-                //wait for VI to connect
+            String dongleId = "trace-playback";
+
+            if (getVehicleManager().isViConnected()) {
+                //VI is connected, use MAC address
+                dongleId = getVehicleManager().getVehicleInterfaceDeviceId();
             }
 
-            String dongleId = getVehicleManager().getVehicleInterfaceDeviceId();
             String make = getPreferenceString(R.string.vehicle_make_key);
             String model = getPreferenceString(R.string.vehicle_model_key);
             String year = getPreferenceString(R.string.vehicle_year_key);
 
             try {
-                VehicleManager vehicleManager = getVehicleManager();
```

Comment:
it was left over from when I used to get the dongleID inline when creating the sink.  just cleaning up